### ISSUE TITLE
Live Coding Environment

### ISSUE THREAD

**Issue by HaoZeke** [symengine/symengine.github.io#12]
Bit of a stretch goal.

**Comment by isuruf** [comment]
We already have one at https://mybinder.org/v2/gh/symengine/symengine/master?filepath=notebooks%2Fsymengine.ipynb

**Comment by HaoZeke** [comment]
Yup, but I was thinking more along the lines of the Sympy shell.
![image](https://user-images.githubusercontent.com/4336207/99717953-03197280-2aa2-11eb-923b-a8fad361eb9d.png)
